Formula/v/vineyard.rb

```diff
@@ -6,7 +6,7 @@ class Vineyard < Formula
   url "https://github.com/v6d-io/v6d/releases/download/v0.23.2/v6d-0.23.2.tar.gz"
   sha256 "2a2788ed77b9459477b3e90767a910e77e2035a34f33c29c25b9876568683fd4"
   license "Apache-2.0"
-  revision 3
+  revision 4
```

Comment:
```suggestion
  revision 3
```
Let's skip the `revision` bump here -- we don't need all the users of this formula to install an essentially identical version with `brew upgrade`.